cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -107,6 +109,88 @@ def test_errored_course_global_staff(self):
             courses_list_by_groups, __ = _accessible_courses_list_from_groups(self.request)
             self.assertEqual(courses_list_by_groups, [])
 
+    @ddt.data(ModuleStoreEnum.Type.split, ModuleStoreEnum.Type.mongo)
+    def test_staff_course_listing_performance(self, default_store):
```

Comment:
Right, We can check for mongo calls. 
Secondly, we can not do that for everyone, we did for global staff because they have access to every course. Other thing is, We are not loading the course, so we are not filtering `ErrorDescriptor` instances.